specification/app/resource-manager/Microsoft.App/preview/2022-01-01-preview/ContainerApps.json

```diff
@@ -309,6 +309,62 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/containerApps/{containerAppName}/analyzeCustomHostName": {
```

Comment:
>analyzeCustomHostName [](http://example.com/codeflow?start=129&length=21)

any reason you are not choosing to name with list* something like .. listCustomHostProperties or something .



The reason I ask is since in ARM templates only list* operations are supported so you could do operations using the output values of this call.

Reply:
See examples here : [https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions-resource#list](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions-resource#list)

Reply:
@raosuhas For context: This API will check if a certain hostname (e.g., my-app.com) is valid for the app by performing a number of checks (e.g., hostname not being used by another app, customer owns the domain, etc.) and returning the results. I think analyze... describes better what the API actually does but I can see the merit on having it available for ARM templates. @ruslany what do you think?